src/DotNetCore.CAP.AzureServiceBus/Consumer/ServiceBusConsumerDescriptorBuilder.cs

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) .NET Core Community. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Azure.Core;
+
+namespace DotNetCore.CAP.AzureServiceBus.Consumer;
+
+public class ServiceBusConsumerDescriptorBuilder
+{
+    /// <summary>
+    /// Name of the group that the consumer is part of.
+    /// </summary>
+    private string GroupName { get; set; } = null!;
+    
+    private string TopicPath { get; set; } = null!;
+    
+    private AzureServiceBusOptions Options { get; set; } = new ();
+    
+    private bool DefaultOptions { get; set; } = false;
+    
+    public ServiceBusConsumerDescriptorBuilder(string groupName)
+    {
+        GroupName = groupName;
+    }
+
+    public ServiceBusConsumerDescriptorBuilder UseTopic(string topicPath)
+    {
+        TopicPath = topicPath;
+        return this;
+    }
+    
+    public ServiceBusConsumerDescriptorBuilder Configuration(Action<AzureServiceBusOptionsBase> configure)
```

Comment:
What do you think of keeping the standard here and calling this method `UseConfiguration` instead?